ambry-router/src/main/java/com.github.ambry.router/PutManager.java

```diff
@@ -259,24 +290,28 @@ void completePendingOperations() {
    * by the {@link ReadableStreamChannel} associated with the operation.
    */
   private class ChunkFiller implements Runnable {
-    private final int sleepTimeWhenIdleMs = 10;
-
     public void run() {
       try {
         while (isOpen.get()) {
-          boolean allChunksFilled = true;
+          boolean idleIteration = true;
           for (PutOperation op : putOperations) {
             if (!op.isChunkFillComplete()) {
               op.fillChunks();
-              allChunksFilled = false;
+              idleIteration = false;
```

Comment:
It seems that the logic of using `idleIteration`is, because this loop there is nothing for me to do, so I will assume the next loop there will also be nothing to do, so I will go to sleep for the next loop. Do we have an assumption like this? Curious if the performance would be substantially impacted if `idleIteration` is removed.